go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/Azure/azure-sdk-for-go v66.0.0+incompatible
 	github.com/Azure/go-autorest/autorest v0.11.28
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
-	github.com/ClickHouse/clickhouse-go/v2 v2.2.0
+	github.com/ClickHouse/clickhouse-go v1.5.1
```

Comment:
please make sure that telemetry still works.

Reply:
If it doesn't work I'm almost sure that golang let's us to use two versions of the same library.

Reply:
Probably I need some help with that, I didn't work with telemetry yet. We cannot use v2 with 1.5.1, cuz v2 requires 1.5.4 which breaks existing code in qan-api.

Reply:
BTW, what is an effort to migrate QAN to v2?

Reply:
https://www.percona.com/blog/2020/03/09/using-different-versions-of-a-package-in-an-application-via-go-modules/

Reply:
> BTW, what is an effort to migrate QAN to v2?

I tried to bump to v2 in QAN, but tests did not pass. We need a separate task for that.

Reply:
I'd say, breaking telemetry is not an option, so not much choice but to use v2 here :)

Reply:
Reversed